educator/tests/Test/Dscp/Educator/Certificates.hs

```diff
@@ -0,0 +1,10 @@
+module Test.Dscp.Educator.Certificates where
+
+import Dscp.Educator
+import Dscp.Util.Test
+
+spec_Educator_certificates :: Spec
+spec_Educator_certificates = do
+    it "Can build a full student certificate flawlessly" . property $
+        \cert faircv -> ioProperty $
+            total <$> buildCertificatePDF cert faircv
```

Comment:
This test fails on CI because template files are not found, see another comment.